Marlin/ultralcd.cpp

```diff
@@ -1870,6 +1903,127 @@ static void lcd_status_screen() {
 
   #endif //SDSUPPORT
 
+  #if ENABLED(LCD_INFO_MENU)
+    #ifdef PRINTCOUNTER
+      /**
+       *
+       * Printer Info > Stastics submenu
+       *
+       */
+      static void lcd_info_stats_menu() {
+        PrintCounter print_job_counter = PrintCounter();
+        print_job_counter.loadStats();
+        printStatistics stats = print_job_counter.getStats();
+
+        if (LCD_CLICKED) lcd_goto_previous_menu(true);
+        START_MENU();
+        STATIC_ITEM(MSG_INFO_TOTAL_PRINTS ": "   ); // Total Prints:
+        STATIC_ITEM(stats.totalPrints            ); // 999
+        STATIC_ITEM(MSG_INFO_FINISHED_PRINTS ": "); // Finished Prints:
+        STATIC_ITEM(stats.finishedPrints         ); // 666
+        STATIC_ITEM(MSG_INFO_PRINT_TIME ": "     ); // Total Print Time:
+        STATIC_ITEM(stats.printTime              ); // 123456
+        END_MENU();
+      }
+    #endif
+  
+    /**
+     *
+     * Printer Info > Thermistors
+     *
+     */
+    static void lcd_info_thermistors_menu() {
+	  if (LCD_CLICKED) lcd_goto_previous_menu(true);
+      START_MENU();
+	  #define THERMISTOR_ID TEMP_SENSOR_0
+      #include "thermistornames.h"
+      STATIC_ITEM("T0: " THERMISTOR_NAME);
+      STATIC_ITEM("    " MSG_INFO_MIN_TEMP ": " STRINGIFY(HEATER_0_MINTEMP));
+      STATIC_ITEM("    " MSG_INFO_MAX_TEMP ": " STRINGIFY(HEATER_0_MAXTEMP));
+
+      #if TEMP_SENSOR_1 != 0
+        #undef THERMISTOR_ID
+        #define THERMISTOR_ID TEMP_SENSOR_1
+        #include "thermistornames.h"
+        STATIC_ITEM("T1: " THERMISTOR_NAME);
+        STATIC_ITEM("    " MSG_INFO_MIN_TEMP ": " STRINGIFY(HEATER_1_MINTEMP));
+        STATIC_ITEM("    " MSG_INFO_MAX_TEMP ": " STRINGIFY(HEATER_1_MAXTEMP));
+      #endif
+
+      #if TEMP_SENSOR_2 != 0
+        #undef THERMISTOR_ID
+        #define THERMISTOR_ID TEMP_SENSOR_2
+        #include "thermistornames.h"
+        STATIC_ITEM("T2: " THERMISTOR_NAME);
+        STATIC_ITEM("    " MSG_INFO_MIN_TEMP ": " STRINGIFY(HEATER_2_MINTEMP));
```

Comment:
The screen width is 20 characters for most, but we need to accommodate a screen width of 18 also.